src/main/kotlin/org/rust/lang/core/psi/ext/RsLoopOnlyExpr.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi.ext
+
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RsExpr
+
+// Expr that can only be used inside a `loop` or `while` block. E.g. `break`, `continue`
+interface RsLoopOnlyExpr : RsExpr, RsLabelReferenceOwner, RsOuterAttributeOwner {
+    val element: PsiElement
```

Comment:
It's quite difficult to understand that is this element without proper name and documentation

Reply:
I agree with you that the name is somewhat misleading. I'm not sure what names should I use though. I have a couple ideas: `RsLoopNeedingExpr`, `RsLoopLikingExpr`, `RsLoopContainedExpr`. I'm leaning towards `RsLoopLikingExpr` as I believe it's the most truthful name but it doesn't really reflect the meaning. Do you have any suggestions? Alternatively, I could just remove this interface and make `RsErrorAnnotator` use type checks.

Reply:
Actually, we already have interface for `break`/`continue` expressions: `RsLabelReferenceOwner`. Yeah, its name is only about labels but it was created to make common interface only for `break`/`continue`.
So let's reuse it.
About naming:
* I can come up with only `RsControlFlowExpr` as an interface name.
* As a better name for `element` property, I can suggest `operator` or something similar

Reply:
Thanks, I have been completely oblivious of `RsLabelReferenceOwner`. I guess creating the `RsLoopOnlyExpr` was completely unnecessary. I have now removed it and replaced it with label reference owner. Additionally, I've added an `operator` property to it so that errors can be displayed on the operator only and not the whole expression. (`break` highlight instead of `break 'foo`).